titus_isolate/event/kubernetes_opportunistic_window_publisher.py

```diff
@@ -51,41 +60,68 @@ def __get_node(self) -> V1Node:
         log.debug('node: %s', node)
         return node
 
+    def __watch(self):
+        label_selector = "{}={}".format(OPPORTUNISTIC_RESOURCE_NODE_NAME_LABEL_KEY,
+                                        self.__node_name)
+        while True:
+            log.info("Starting opportunistic resource watch...")
+            try:
+                stream = watch.Watch().stream(
+                    self.__custom_api.list_cluster_custom_object,
+                    "titus.netflix.com",
+                    "v1",
+                    "opportunistic-resources",
+                    label_selector=label_selector)
+
+                for event in stream:
+                    log.info("Event: %s", event)
+                    event_type = event['type']
+                    event_metadata_name = event['object']['metadata']['name']
+
+                    with self.__lock:
+                        if event_type == 'ADDED':
+                            self.__opportunistic_resources[event_metadata_name] = event
+                        elif event_type == 'DELETED':
+                            self.__opportunistic_resources.pop(event_metadata_name, None)
+
+            except:
```

Comment:
Can you do `except Exception` instead of the bare except?
https://stackoverflow.com/a/54948581

Reply:
yes, done